### ISSUE TITLE
Dynamic layout problems with First Law panels

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/keplers-laws#130]
Related to https://github.com/phetsims/keplers-laws/issues/86#issuecomment-1696022288 and #122 ...

The fraction line in one of the First Law panels does not dynamically resize.  For example:

<img width="334" alt="screenshot_2735" src="https://github.com/phetsims/keplers-laws/assets/3046552/f4c2694a-5cee-4ba4-88d3-8110a61667c2">

**Comment by pixelzoom** [comment]
I did this one. @AgustinVallejo please review, close if OK.

**Comment by pixelzoom** [comment]
I realized that we also needed to address FirstLawMoreDataPanel, so I did that in the above commit.  But there's still layout problem with the InfoButton overlapping the top line of text:

<img width="338" alt="screenshot_2746" src="https://github.com/phetsims/keplers-laws/assets/3046552/15c6bd92-f72d-4db4-9bc1-8fbf0e4e814b">

**Comment by pixelzoom** [comment]
In the above commit, I fixed the overlap, see screenshot below.   I was simple, just had to add missing `spacing: 5` to the HBox, see https://github.com/phetsims/keplers-laws/commit/637e35bf5669b8e8a49583723be8f17aab1fe957.  

<img width="346" alt="screenshot_2748" src="https://github.com/phetsims/keplers-laws/assets/3046552/e574bd8f-eb62-4536-9f65-53c4b0364d9e">

But this simple fix was so hidden in the nesting of HBoxes and VBoxes that I couldn't see it until I reorganized the code in https://github.com/phetsims/keplers-laws/commit/dcb95eb0633b76d5f024735d6d7ce64a301f5598.  There was also this code duplicated 5x, which I cleaned up by moving the HBox creation into `createCustomEquation`, and eliminating the unnecessary `visibleProperty`:

```ts
            new HBox( {
              spacing: 2,
              visibleProperty: KeplersLawsPreferences.moreOrbitalDataEnabledProperty,
              children: createCustomEquation(
                KeplersLawsStrings.symbols.positionMagnitudeStringProperty,
                positionMagnitudeStringProperty,
                KeplersLawsColors.distancesColorProperty
              )
            } ),
```

This is exactly the problem that I mentioned in https://github.com/phetsims/keplers-laws/issues/112 (Heavily-nested control panels are difficult to read and instrument).   It made this issue take 5x as long as it should have. 

Ready for @AgustinVallejo to review. Close if OK.
